outputs/cloudevents.go

```diff
@@ -0,0 +1,66 @@
+package outputs
+
+import (
+	"context"
+	"log"
+	"strings"
+
+	cloudevents "github.com/cloudevents/sdk-go/v2"
+
+	"github.com/falcosecurity/falcosidekick/types"
+)
+
+// CloudEvent produces a CloudEvent and sends to the CloudEvents consumers.
+func (c *Client) CloudEvents(falcopayload types.FalcoPayload) {
+	c.Stats.CloudEvents.Add(Total, 1)
+
+	if c.ce == nil {
+		client, err := cloudevents.NewDefaultClient()
+		if err != nil {
+			go c.CountMetric(Outputs, 1, []string{"output:cloudevents", "status:error"})
+			log.Println("[ERROR] : CloudEvents - NewDefaultClient", err)
+			return
+		}
+		c.ce = client
+	}
+
+	ctx := cloudevents.ContextWithTarget(context.Background(), c.EndpointURL.String())
+
+	event := cloudevents.NewEvent()
+	event.SetTime(falcopayload.Time)
+	event.SetSource("falco.org") // TODO: this should have some info on the falco server that made the event.
```

Comment:
```suggestion
	event.SetSource("falco") // TODO: this should have some info on the falco server that made the event.
```

`falco` doesn't send this intel (`host` or else) in its output.

Reply:
If it did, you would know which falco service has made this observation.

Reply:
what do you call `falco service`?

Reply:
Ideally, you can find the falco install using a combo of `cloudevent.source` and `cloudevents.subject`

Reply:
source is suppose to be a uri-ref...

Reply:
maybe change it in a followup?

Reply:
If it's supposed to be an URI, ok for `falco.org`